### PR TITLE
Fix blazegraph permission issues

### PR DESCRIPTION
**Pull request by nfusionz**
Some work still needs to be done for the initial startup, but `redis` should no longer overwrite the permissions of the `data` folder.

### CODE REVIEW

File: docs/docker.md [hunk 1]
```diff
@@ -77,6 +77,8 @@ The monolithic image was split into images for the databases Whyis relies on (_r
 #### Prerequisites 
 * Docker
 * `docker-compose`
+* A data folder in the same directory as your app and Whyis with permissions 777
```
Comment (minorg) [inline docs/docker.md]: should be /data alongside /apps, so I don't think this is right.

File: docs/docker.md [hunk 2]
```diff
@@ -102,6 +104,16 @@ Further, `docker/compose/split/app-dev` references a version of `whyis-server` t
 
 ## Other notes
 
+### Data directory
+
+By default, the split `docker-compose.yml` will mount a directory named `data` in the same directory as your Whyis folder on the host machine. Due to how Docker allocates volumes, this folder will have the wrong permissions and cause Blazegraph to fail on the first setup. To avoid this, run the following instructions before running `docker-compose`:
```
Comment (minorg) [inline docs/docker.md]: I don't think this is right, either. Look at the relative path.

File: docs/docker.md (hunk 1 above)
Comment (nfusionz) [reply docs/docker.md]: You're right, just checked.

**Comment by minorg** [comment]
I'm on my personal user. Please consider this approved. Good work, Owen.

**Pull request merged**